---
layout: about
title: about
permalink: /
subtitle: #<a href='#'>Affiliations</a>. Address. Contacts. Motto. Etc.

# profile:
#   align: center
#   image: ceel_logo.png
#   image_circular: false # crops the image to make it circular
#   # more_info: >
#   #   <p>555 your office number</p>
#   #   <p>123 your address street</p>
#   #   <p>Your City, State 12345</p>

selected_papers: true # includes a list of papers marked as "selected={true}"
#social: true # includes social icons at the bottom of the page

announcements:
  enabled: true # includes a list of news items
  scrollable: true # adds a vertical scroll bar if there are more than 3 news items
  limit: 5 # leave blank to include all the news in the `_news` folder

# latest_posts:
#   enabled: false
#   scrollable: true # adds a vertical scroll bar if there are more than 3 new posts items
#   limit: 3 # leave blank to include all the blog posts
---

<p align="center">
  <img src="/assets/img/ceel_logo.png" alt="Logo" width="500">
</p>

# Evolution of Metabolic Phenotypes in Variable Tumor Microenvironments

The primary aim of my laboratory is to apply ecological and evolutionary principles to elucidate the molecular mechanisms driving the initiation, progression, and metastasis of breast and ovarian cancers through adaptation to their microenvironments. I hypothesize that the evolution of novel metabolic phenotypes under variable microenvironmental selection pressures can promote cancer development via metabolic reprogrammingâ€”linked to epigenetic modifications as well as the activation and stabilization of transcription factors.

Our research focuses on how tumor adaptation remodels the microenvironment and metabolism, creating new selective pressures. This dynamic interplay between tumor cells and their surrounding stroma is fundamental to the development of a constantly evolving tumor ecosystem, which in turn leads to increased genotypic heterogeneity and phenotypic plasticity. To capture this complexity, we integrate single-cell analyses across the genome, epigenome, transcriptome, proteome, metabolome, and pathomics.