<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | CEEL: Cancer Ecology &amp; Evolution Lab </title> <meta name="author" content="CEEL : Cancer Ecology &amp; Evolution Lab"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://sbu-damaghi-ceel.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> CEEL: Cancer Ecology &amp; Evolution Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">search <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Xiao2024-ty" class="col-sm-10"> <div class="title">Eco-evolutionary guided pathomic analysis to predict DCIS upstaging</div> <div class="author"> Yujie Xiao, Manal Elmasry, Ji Dong K Bai, Andrew Chen, Yuzhu Chen, Brooke Jackson, Joseph O Johnson, Robert J Gillies, Prateek Prasanna, Chao Chen, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>bioRxivorg</em>, Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cancers evolve in a dynamic ecosystem. Thus, characterizing cancer’s ecological dynamics is crucial to understanding cancer evolution and can lead to discovering novel biomarkers to predict disease progression. Ductal carcinoma in situ (DCIS) is an early-stage breast cancer characterized by abnormal epithelial cell growth confined within the milk ducts. In this study, we show that ecological habitat analysis of hypoxia and acidosis biomarkers can significantly improve prediction of DCIS upstaging. First, we developed a novel eco-evolutionary designed approach to define habitats in the tumor intraductal microenvironment based on oxygen diffusion distance. Then, we identify cancer cells with metabolic phenotypes attributed to their habitat conditions, such as the expression of CA9 indicating hypoxia responding phenotype, and LAMP2b indicating the acid adaptation. Traditionally these markers have shown limited predictive capabilities for DCIS upstaging, if any. However, when analyzed from an ecological perspective, their power to differentiate between pure DCIS and upstaged DCIS increased significantly. Second, using eco-evolutionary guided computational and digital pathology techniques, we discovered distinct niches with spatial patterns of these biomarkers and used the distribution of such niches to predict patient upstaging. The niches patterns were characterized by pattern analysis of both cellular and spatial features. With a 5-fold validation on the biopsy cohort, we trained a random forest classifier to achieve the area under curve (AUC) of 0.74. Our results affirm the importance of using eco-evolutionary-designed approaches in biomarkers discovery studies in the era of digital pathology by demonstrating the role of tumor ecological habitats and niches.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Song2024-zc" class="col-sm-10"> <div class="title">Human and bats genome robustness under COSMIC mutational signatures</div> <div class="author"> Joon-Hyun Song, Ying Zeng, Liliana M Dávalos, Thomas MacCarthy, Mani Larijani, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>bioRxivorg</em>, Sep 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Han2024-bp" class="col-sm-10"> <div class="title">Ecological and evolutionary dynamics to design and improve ovarian cancer treatment</div> <div class="author"> Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>Clin. Transl. Med.</em>, Sep 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP-ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell’s plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. KEY POINTS/HIGHLIGHTS: Tumours are ecosystems in which cancer and non-cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco-evolutionarily designed therapies should consider cancer cell plasticity and patient-specific characteristics to improve clinical outcome and prevent relapse.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Strobl2024-zb" class="col-sm-10"> <div class="title">To modulate or to skip: De-escalating PARP inhibitor maintenance therapy in ovarian cancer using adaptive therapy</div> <div class="author"> Maximilian A R Strobl, Alexandra L Martin, Jeffrey West, Jill Gallaher, Mark Robertson-Tessi, Robert Gatenby, Robert Wenham, Philip K Maini, <em>Mehdi Damaghi</em>, and Alexander R A Anderson </div> <div class="periodical"> <em>Cell Syst.</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Toxicity and emerging drug resistance pose important challenges in poly-adenosine ribose polymerase inhibitor (PARPi) maintenance therapy of ovarian cancer. We propose that adaptive therapy, which dynamically reduces treatment based on the tumor dynamics, might alleviate both issues. Utilizing in vitro time-lapse microscopy and stepwise model selection, we calibrate and validate a differential equation mathematical model, which we leverage to test different plausible adaptive treatment schedules. Our model indicates that adjusting the dosage, rather than skipping treatments, is more effective at reducing drug use while maintaining efficacy due to a delay in cell kill and a diminishing dose-response relationship. In vivo pilot experiments confirm this conclusion. Although our focus is toxicity mitigation, reducing drug use may also delay resistance. This study enhances our understanding of PARPi treatment scheduling and illustrates the first steps in developing adaptive therapies for new treatment settings. A record of this paper’s transparent peer review process is included in the supplemental information.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Song2024-dy" class="col-sm-10"> <div class="title">Evolvability of cancer-associated genes under APOBEC3A/B selection</div> <div class="author"> Joon-Hyun Song, Liliana M Dávalos, Thomas MacCarthy, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>iScience</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Evolvability is an emergent hallmark of cancer that depends on intra-tumor heterogeneity and genetic variation. Mutations generated by APOBEC3 contribute to genetic variation and tumor evolvability. However, the influence of APOBEC3 on the evolvability of the genome and its differential impact on cancer genes versus non-cancer genes remains unclear. Analyzing over 40,000 human protein-coding transcripts, we identified distinct distribution patterns of APOBEC3A/B TC motifs between cancer and non-cancer genes, suggesting unique associations with cancer. Studying a bat species with numerous APOBEC3 genes, we found distinct motif patterns in orthologs of cancer genes compared to non-cancer genes, as in humans, suggesting APOBEC3 evolution to reduce impacts on the genome rather than the converse. Simulations confirmed that APOBEC3-induced heterogeneity enhances cancer evolution through bimodal patterns of mutations in certain classes of genes. Our results suggest the bimodal distribution of APOBEC-induced mutations can significantly increase cancer heterogeneity.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Obeid2024-ju" class="col-sm-10"> <div class="title">Metabolic adaptation and epigenetic modulations: Unraveling tumor plasticity under variable tumor microenvironment</div> <div class="author"> Jowana Obeid, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em></em> Jan 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The tumor microenvironment (TME) variation plays a crucial role in cancer initiation, progression, and evolution. Tumor plasticity, defined as the ability of cancer cells to adapt their variable TME with no need for mutations or genetic changes, has emerged as a hallmark of cancer driving tumor evolution. This chapter discusses the metabolic adaptations as part of tumor plasticity to variable TME that occur in solid tumors, affecting the tumor evolutionary trajectories. It highlights the role of epigenetic modifications in adaptation to variable TME, including DNA methylation, chromatin modifications, and noncoding RNA regulation. Additionally, it explores how the TME conditions cause reprograming cellular metabolism, including glucose, amino acid, and lipid metabolism, leading to major rewiring of transcriptional networks and accumulation of reactive oxygen species (ROS) that promote tumor progression. The interplay …</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Poonja2023-do" class="col-sm-10"> <div class="title">Dynamics of fibril collagen remodeling by tumor cells: A model of tumor-associated collagen signatures</div> <div class="author"> Sharan Poonja, Ana Forero Pinto, Mark C Lloyd, <em>Mehdi Damaghi</em>, and K Rejniak </div> <div class="periodical"> <em>Cells</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Many solid tumors are characterized by a dense extracellular matrix (ECM) composed of various ECM fibril proteins. These proteins provide structural support and a biological context for the residing cells. The reciprocal interactions between growing and migrating tumor cells and the surrounding stroma result in dynamic changes in the ECM architecture and its properties. With the use of advanced imaging techniques, several specific patterns in the collagen surrounding the breast tumor have been identified in both tumor murine models and clinical histology images. These tumor-associated collagen signatures (TACS) include loosely organized fibrils far from the tumor and fibrils aligned either parallel or perpendicular to tumor colonies. They are correlated with tumor behavior, such as benign growth or invasive migration. However, it is not fully understood how one specific fibril pattern can be dynamically remodeled to form another alignment. Here, we present a novel multi-cellular lattice-free (MultiCell-LF) agent-based model of ECM that, in contrast to static histology images, can simulate dynamic changes between TACSs. This model allowed us to identify the rules of cell–ECM physical interplay and feedback that guided the emergence and transition among various TACSs.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Song2023-ot" class="col-sm-10"> <div class="title">Evolvability of cancer-associated genes under APOBEC3A/B selection</div> <div class="author"> Joon-Hyun Song, Liliana M Dávalos, Thomas MacCarthy, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>bioRxiv</em>, Aug 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Evolvability is an emergent hallmark of cancer that depends on intra-tumor heterogeneity and, ultimately, genetic variation. Mutations generated by APOBEC3 cytidine deaminases can contribute to genetic variation and the consequences of APOBEC activation differ depending on the stage of cancer, with the most significant impact observed during the early stages. However, how APOBEC activity shapes evolutionary patterns of genes in the host genome and differential impacts on cancer-associated and non-cancer genes remain unclear. Analyzing over 40,000 human protein-coding transcripts, we identified distinct distribution patterns of APOBEC3A/B TC motifs between cancer-related genes and controls, suggesting unique associations with cancer. Studying a bat species with many more APOBEC3 genes, we found diverse motif patterns in orthologs of cancer genes compared to controls, similar to humans and suggesting APOBEC evolution to reduce impacts on the genome rather than the converse. Simulations confirmed that APOBEC-induced heterogeneity enhances cancer evolution, shaping clonal dynamics through bimodal introduction of mutations in certain classes of genes. Our results suggest that a major consequence of the bimodal distribution of APOBEC affects greater cancer heterogeneity. Highlights ### Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Strobl2023-rp" class="col-sm-10"> <div class="title">Adaptive therapy for ovarian cancer: An integrated approach to PARP inhibitor scheduling</div> <div class="author"> Maximilian Strobl, Alexandra L Martin, Jeffrey West, Jill Gallaher, Mark Robertson-Tessi, Robert Gatenby, Robert Wenham, Philip Maini, <em>Mehdi Damaghi</em>, and Alexander Anderson </div> <div class="periodical"> <em>bioRxivorg</em>, Mar 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Toxicity and emerging drug resistance are important challenges in PARP inhibitor (PARPi) treatment of ovarian cancer. Recent research has shown that evolutionary-inspired treatment algorithms which adapt treatment to the tumorâ€™s treatment response (adaptive therapy) can help to mitigate both. Here, we present a first step in developing an adaptive therapy protocol for PARPi treatment by combining mathematical modelling and wet-lab experiments to characterize the cell population dynamics under different PARPi schedules. Using data from in vitro Incucyte Zoom time-lapse microscopy experiments and a step-wise model selection process we derive a calibrated and validated ordinary differential equation model, which we then use to test different plausible adaptive treatment schedules. Our model can accurately predict the in vitro treatment dynamics, even to new schedules, and suggests that treatment modifications need to be carefully timed, or one risks losing control over tumour growth, even in the absence of any resistance. This is because our model predicts that multiple rounds of cell division are required for cells to acquire sufficient DNA damage to induce apoptosis. As a result, adaptive therapy algorithms that modulate treatment but never completely withdraw it are predicted to perform better in this setting than strategies based on treatment interruptions. Pilot experiments in vivo confirm this conclusion. Overall, this study contributes to a better understanding of the impact of scheduling on treatment outcome for PARPis and showcases some of the challenges involved in developing adaptive therapies for new treatment settings.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Rodriguez-Messan2021-sd" class="col-sm-10"> <div class="title">Predicting the results of competition between two breast cancer lines grown in 3-D spheroid culture</div> <div class="author"> Marisabel Rodriguez Messan, <em>Mehdi Damaghi</em>, Audrey Freischel, Yan Miao, Joel Brown, Robert Gillies, and Dorothy Wallace </div> <div class="periodical"> <em>Math. Biosci.</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>This study develops a novel model of a consumer-resource system with mobility included, in order to explain a novel experiment of competition between two breast cancer cell lines grown in 3D in vitro spheroid culture. The model reproduces observed differences in monoculture, such as overshoot phenomena and final size. It also explains both theoretically and through simulation the inevitable triumph of the same cell line in co-culture, independent of initial conditions. The mobility of one cell line (MDA-MB-231) is required to explain both the success and the rapidity with which that species dominates the population and drives the other species (MCF-7) to extinction. It is shown that mobility directly interferes with the other species and that the cost of that mobility is in resource usage rate.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Poonja2021-ei" class="col-sm-10"> <div class="title">Dynamics of fibril collagen remodeling by tumor cells using individual cell-based mathematical modeling</div> <div class="author"> Sharan Poonja, <em>Mehdi Damaghi</em>, and Katarzyna A Rejniak </div> <div class="periodical"> <em>bioRxiv</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>AbstractMany solid tumors are characterized by dense extracellular matrix (ECM) composed of various ECM fibril proteins that provide structural support and biological context for the residing cells. The growing tumor cell colonies are capable of remodeling the ECM structure in tumor immediate vicinity to form specific microenvironmental niches. The changes in fibril patterns of the collagen (one of the ECM proteins) surrounding the tumor can be visualized experimentally using both histology and fluorescent imaging. In particular, three diverse tumor associated collagen signatures (TACS) were identified and related to tumor behavior, such as benign growth or invasion. Here we will use an off-lattice hybrid agent-based model (MultiCell-LF) to identify the rules of cell-ECM interactions that guide the development of various patterns of alignment of the ECM fibrils.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ordway2021-ip" class="col-sm-10"> <div class="title">Extracellular acidification induces lysosomal dysregulation</div> <div class="author"> Bryce Ordway, Robert J Gillies, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em></em> May 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Many invasive cancers emerge through a years-long process of somatic evolution, characterized by an accumulation of heritable genetic and epigenetic changes and the emergence of increasingly aggressive clonal populations. In solid tumors, such as breast ductal carcinoma, the extracellular environment for cells within the nascent tumor is harsh and imposes different types of stress on cells, such as hypoxia, nutrient deprivation, and cytokine inflammation. Acidosis is a constant stressor of most cancer cells due to its production through fermentation of glucose to lactic acid in hypoxic or normoxic regions (Warburg effect). Over a short period of time, acid stress can have a profound effect on the function of lysosomes within the cells exposed to this environment, and after long term exposure, lysosomal function of the cancer cells can become completely dysregulated. Whether this dysregulation is due to an epigenetic change or evolutionary selection has yet to be determined, but understanding the mechanisms behind this dysregulation could identify therapeutic opportunities.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Kimmel2021-vl" class="col-sm-10"> <div class="title">Local contact inhibition leads to universal principles of cell population growth</div> <div class="author"> Gregory J Kimmel, Jeffrey West, <em>Mehdi Damaghi</em>, Alexander R A Anderson, and Philipp M Altrock </div> <div class="periodical"> <em>arXiv [q-bio.PE]</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cancer cell population dynamics often exhibit remarkably replicable, universal laws despite their underlying heterogeneity. Mechanistic explanations of universal cell population growth remain partly unresolved to this day, whereby population feedback between the microscopic and mesoscopic configurations can lead to macroscopic growth laws. We here present a unification under density-dependent birth events via contact inhibition. We consider five classical tumor growth laws: exponential, generalized logistic, Gompertz, radial growth, and fractal growth, which can be seen as manifestations of a single microscopic model. Our theory is substantiated by agent based simulations and can explain growth curve differences in experimental data from in vitro cancer cell population growth. Thus, our framework offers a possible explanation for the large number of mean-field laws that can adequately capture seemingly unrelated cancer or microbial growth dynamics.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Jalili2021-dv" class="col-sm-10"> <div class="title">Exploring the metabolic heterogeneity of cancers: A benchmark study of context-specific models</div> <div class="author"> M Jalili, Martin Scharm, O Wolkenhauer, <em>Mehdi Damaghi</em>, and Ali Salehzadeh-Yazdi </div> <div class="periodical"> <em>J. Pers. Med.</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Metabolic heterogeneity is a hallmark of cancer and can distinguish a normal phenotype from a cancer phenotype. In the systems biology domain, context-specific models facilitate extracting physiologically relevant information from high-quality data. Here, to utilize the heterogeneity of metabolic patterns to discover biomarkers of all cancers, we benchmarked thousands of context-specific models using well-established algorithms for the integration of omics data into the generic human metabolic model Recon3D. By analyzing the active reactions capable of carrying flux and their magnitude through flux balance analysis, we proved that the metabolic pattern of each cancer is unique and could act as a cancer metabolic fingerprint. Subsequently, we searched for proper feature selection methods to cluster the flux states characterizing each cancer. We employed PCA-based dimensionality reduction and a random forest learning algorithm to reveal reactions containing the most relevant information in order to effectively identify the most influential fluxes. Conclusively, we discovered different pathways that are probably the main sources for metabolic heterogeneity in cancers. We designed the GEMbench website to interactively present the data, methods, and analysis results.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Khajehdehi2021-ri" class="col-sm-10"> <div class="title">Long noncoding RNAs in gastrointestinal cancer: Tumor suppression versus tumor promotion</div> <div class="author"> Mina Khajehdehi, Mohammad Khalaj-Kondori, Tayyebeh Ghasemi, Babak Jahanghiri, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>Dig. Dis. Sci.</em>, Feb 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Approximately 80% of the human genome harbors biochemical marks of active transcription that its majority transcribes to noncoding RNAs, namely long noncoding RNAs (lncRNAs). LncRNAs are heterogeneous RNA transcripts that regulate critical biological processes such as cell survival and death. They involve in the progression of different cancers by affecting transcriptional and post-transcriptional modifications as well as epigenetic control of numerous tumor suppressors and oncogenes. Recent findings show that aberrant expression of lncRNAs is associated with tumor initiation, progression, invasion, and overall survival of patients with gastrointestinal (GI) cancers. Some lncRNAs play as tumor suppressors in all GI cancers, but others play as tumor promoters. However, some other lncRNAs might function as a tumor suppressor in one GI cancer, but as a tumor promoter in another GI cancer type. This fact highlights possible context dependency of the expression patterns and roles of at least some lncRNAs in GI cancer development and progression. Here, we review the functional relation of lncRNAs involved in the development and progression of GI cancer by focusing on their roles as tumor suppressor and tumor promoter genes.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Sadeghi2020-ap" class="col-sm-10"> <div class="title">Integrative analysis of breast cancer cells reveals an epithelial-mesenchymal transition role in adaptation to acidic microenvironment</div> <div class="author"> Mehdi Sadeghi, Bryce Ordway, Ilyia Rafiei, Punit Borad, Bin Fang, John L Koomen, Chaomei Zhang, Sean Yoder, Joseph Johnson, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>Front. Oncol.</em>, Mar 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Early ducts of breast tumors are unequivocally acidic. High rates of glycolysis combined with poor perfusion lead to a congestion of acidic metabolites in the tumor microenvironment, and pre-malignant cells must adapt to this acidosis to thrive. Adaptation to acidosis selects cancer cells that can thrive in harsh conditions and are capable of outgrowing the normal or non-adapted neighbors. This selection is usually accompanied by phenotypic change. Epithelial mesenchymal transition (EMT) is one of the most important switches correlated to malignant tumor cell phenotype and has been shown to be induced by tumor acidosis. New evidence shows that the EMT switch is not a binary system and occurs on a spectrum of transition states. During confirmation of the EMT phenotype, our results demonstrated a partial EMT phenotype in our acid-adapted cell population. Using RNA sequencing and network analysis we found 10 dysregulated network motifs in acid-adapted breast cancer cells playing a role in EMT. Our further integrative analysis of RNA sequencing and SILAC proteomics resulted in recognition of S100B and S100A6 proteins at both the RNA and protein level. Higher expression of S100B and S100A6 was validated in vitro by Immunocytochemistry. We further validated our finding both in vitro and in patients’ samples by IHC analysis of Tissue Microarray (TMA). Correlation analysis of S100A6 and LAMP2b as marker of acidosis in each patient from Moffitt TMA approved the acid related role of S100A6 in breast cancer patients. Also, DCIS patients with higher expression of S100A6 showed lower survival compared to lower expression. We propose essential roles of acid adaptation in cancer cells EMT process through S100 proteins such as S100A6 that can be used as therapeutic strategy targeting both acid-adapted and malignant phenotypes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ordway2020-wf" class="col-sm-10"> <div class="title">Causes and consequences of variable tumor cell metabolism on heritable modifications and tumor evolution</div> <div class="author"> Bryce Ordway, Pawel Swietach, Robert J Gillies, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>Front. Oncol.</em>, Mar 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>When cancer research advanced into the post-genomic era, it was widely anticipated that the sought-after cure will be delivered promptly. Instead, it became apparent that an understanding of cancer genomics, alone, is unable to translate the wealth of information into successful cures. While gene sequencing has significantly improved our understanding of the natural history of cancer and identified candidates for therapeutic targets, it cannot predict the impact of the biological response to therapies. Hence, patients with a common mutational profile may respond differently to the same therapy, due in part to different microenvironments impacting on gene regulation. This complexity arises from a feedback circuit involving epigenetic modifications made to genes by the metabolic byproducts of cancer cells. New insights into epigenetic mechanisms, activated early in the process of carcinogenesis, have been able to describe phenotypes which cannot be inferred from mutational analyses per se. Epigenetic changes can propagate throughout a tumor via heritable modifications that have long-lasting consequences on ensuing phenotypes. Such heritable epigenetic changes can be evoked profoundly by cancer cell metabolites, which then exercise a broad remit of actions across all stages of carcinogenesis, culminating with a meaningful impact on the tumor’s response to therapy. This review outlines some of the cross-talk between heritable epigenetic changes and tumor cell metabolism, and the consequences of such changes on tumor progression.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ordway2020-kx" class="col-sm-10"> <div class="title">Targeting of evolutionarily acquired cancer cell phenotype by exploiting pHi-metabolic vulnerabilities</div> <div class="author"> Bryce Ordway, M Tomaszewski, S Byrne, D Abrahams, P Swietach, R Gillies, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>Cancers (Basel)</em>, Dec 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Evolutionary dynamics can be used to control cancers when a cure is not clinically considered to be achievable. Understanding Darwinian intratumoral interactions of microenvironmental selection forces can be used to steer tumor progression towards a less invasive trajectory. Here, we approach intratumoral heterogeneity and evolution as a dynamic interaction among subpopulations through the application of small, but selective biological forces such as intracellular pH (pHi) and/or extracellular pH (pHe) vulnerabilities. Increased glycolysis is a prominent phenotype of cancer cells under hypoxia or normoxia (Warburg effect). Glycolysis leads to an important aspect of cancer metabolism: reduced pHe and higher pHi. We recently showed that decreasing pHi and targeting pHi sensitive enzymes can reverse the Warburg effect (WE) phenotype and inhibit tumor progression. Herein, we used diclofenac (DIC) repurposed to control MCT activity, and Koningic acid (KA) that is a GAPDH partial inhibitor, and observed that we can control the subpopulation of cancer cells with WE phenotype within a tumor in favor of a less aggressive phenotype without a WE to control progression and metastasis. In a 3D spheroid co-cultures, we showed that our strategy can control the growth of more aggressive MDA-MB-231 cells, while sparing the less aggressive MCF7 cells. In an animal model, we show that our approach can reduce tumor growth and metastasis. We thus propose that evolutionary dynamics can be used to control tumor cells’ clonal or sub-clonal populations in favor of slower growth and less damage to patients. We propose that this can result in cancer control for tumors where cure is not an option.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Pressley2020-bf" class="col-sm-10"> <div class="title">Cycling hypoxia selects for constitutive HIF stabilization</div> <div class="author"> Mariyah Pressley, J Gallaher, Joel s Brown, M Tomaszewski, P Borad, <em>Mehdi Damaghi</em>, R Gillies, and C Whelan </div> <div class="periodical"> <em>Sci. Rep.</em>, Oct 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Tumors experience temporal and spatial fluctuations in oxygenation. Hypoxia inducible transcription factors (HIF-α) respond to low levels of oxygen and induce re-supply oxygen. HIF-α stabilization is typically facultative, induced by hypoxia and reduced by normoxia. In some cancers, HIF-α stabilization becomes constitutive under normoxia. We develop a mathematical model that predicts how fluctuating oxygenation affects HIF-α stabilization and impacts net cell proliferation by balancing the base growth rate, the proliferative cost of HIF-α expression, and the mortality from not expressing HIF-α during hypoxia. We compare optimal net cell proliferation rate between facultative and constitutive HIF-α regulation in environments with different oxygen profiles. We find that that facultative HIF-α regulation promotes greater net cell proliferation than constitutive regulation with stochastic or slow periodicity in oxygenation. However, cell fitness is nearly identical for both HIF-α regulation strategies under rapid periodic oxygenation fluctuations. The model thus indicates that cells constitutively expressing HIF-α may be at a selective advantage when the cost of expression is low. In cancer, this condition is known as pseudohypoxia or the “Warburg Effect”. We conclude that rapid and regular cycling of oxygenation levels selects for pseudohypoxia, and that this is consistent with the ecological theory of optimal defense.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Freischel2020-tu" class="col-sm-10"> <div class="title">Frequency-dependent interactions determine outcome of competition between two breast cancer cell lines</div> <div class="author"> Audrey R Freischel, <em>Mehdi Damaghi</em>, J Cunningham, Arig Ibrahim-Hashim, R Gillies, R Gatenby, and Joel s Brown </div> <div class="periodical"> <em>Sci. Rep.</em>, Mar 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Tumors are highly dynamic ecosystems in which diverse cancer cell subpopulations compete for space and resources. These complex, often non-linear interactions govern continuous spatial and temporal changes in the size and phenotypic properties of these subpopulations. Because intra-tumoral blood flow is often chaotic, competition for resources may be a critical selection factor in progression and prognosis. Here, we quantify resource competition using 3D spheroid cultures with MDA-MB-231 and MCF-7 breast cancer cells. We hypothesized that MCF-7 cells, which primarily rely on efficient aerobic glucose metabolism, would dominate the population under normal pH and low glucose conditions; and MDA-MB-231 cells, which exhibit high levels of glycolytic metabolism, would dominate under low pH and high glucose conditions. In spheroids with single populations, MCF-7 cells exhibited equal or superior intrinsic growth rates (density-independent measure of success) and carrying capacities (density-dependent measure of success) when compared to MDA-MB-231 cells under all pH and nutrient conditions. Despite these advantages, when grown together, MCF-7 cells do not always outcompete MDA-MB-231 cells. MDA-MB-231 cells outcompete MCF-7 cells in low glucose conditions and coexistence is achieved in low pH conditions. Under all conditions, MDA-MB-231 has a stronger competitive effect (frequency-dependent interaction) on MCF-7 cells than vice-versa. This, and the inability of growth rate or carrying capacity when grown individually to predict the outcome of competition, suggests a reliance on frequency-dependent interactions and the need for competition assays. We frame these results in a game-theoretic (frequency-dependent) model of cancer cell interactions and conclude that competition assays can demonstrate critical density-independent, density-dependent and frequency-dependent interactions that likely contribute to in vivo outcomes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Damaghi2020-wg" class="col-sm-10"> <div class="title">Collagen production and niche engineering: A novel strategy for cancer cells to survive acidosis in DCIS and evolve</div> <div class="author"> <em>Mehdi Damaghi</em>, Hidetoshi Mori, Samantha Byrne, Liping Xu, Tingan Chen, Joseph Johnson, Nathan D Gallant, Andriy Marusyk, Alexander D Borowsky, and Robert J Gillies </div> <div class="periodical"> <em>Evol. Appl.</em>, Dec 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Growing tumors are dynamic and nonlinear ecosystems, wherein cancer cells adapt to their local microenvironment, and these adaptations further modify the environment, inducing more changes. From nascent intraductal neoplasms to disseminated metastatic disease, several levels of evolutionary adaptations and selections occur. Here, we focus on one example of such an adaptation mechanism, namely, “niche construction” promoted by adaptation to acidosis, which is a metabolic adaptation to the early harsh environment in intraductal neoplasms. The avascular characteristics of ductal carcinoma in situ (DCIS) make the periluminal volume profoundly acidic, and cancer cells must adapt to this to survive. Based on discovery proteomics, we hypothesized that a component of acid adaptation involves production of collagen by pre-cancer cells that remodels the extracellular matrix (ECM) and stabilizes cells under acid stress. The proteomic data were surprising as collagen production and deposition are commonly believed to be the responsibility of mesenchymally derived fibroblasts, and not cells of epithelial origin. Subsequent experiments in 3D culture, spinning disk and second harmonic generation microscopy of DCIS lesions in patients’ samples are concordant. Collagen production assay by acid-adapted cells in vitro demonstrated that the mechanism of induction involves the RAS and SMAD pathways. Secretome analyses show upregulation of ECM remodeling enzymes such as TGM2 and LOXL2 that are collagen crosslinkers. These data strongly indicate that acidosis in incipient cancers induces collagen production by cancer cells and support the hypothesis that this adaptation initiates a tumor-permissive microenvironment promoting survival and growth of nascent cancers.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Strobl2020-hq" class="col-sm-10"> <div class="title">Mix and match: Phenotypic coexistence as a key facilitator of cancer invasion</div> <div class="author"> Maximilian A R Strobl, Andrew L Krause, <em>Mehdi Damaghi</em>, Robert Gillies, Alexander R A Anderson, and Philip K Maini </div> <div class="periodical"> <em>Bull. Math. Biol.</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Invasion of healthy tissue is a defining feature of malignant tumours. Traditionally, invasion is thought to be driven by cells that have acquired all the necessary traits to overcome the range of biological and physical defences employed by the body. However, in light of the ever-increasing evidence for geno- and phenotypic intra-tumour heterogeneity, an alternative hypothesis presents itself: could invasion be driven by a collection of cells with distinct traits that together facilitate the invasion process? In this paper, we use a mathematical model to assess the feasibility of this hypothesis in the context of acid-mediated invasion. We assume tumour expansion is obstructed by stroma which inhibits growth and extra-cellular matrix (ECM) which blocks cancer cell movement. Further, we assume that there are two types of cancer cells: (i) a glycolytic phenotype which produces acid that kills stromal cells and (ii) a matrix-degrading phenotype that locally remodels the ECM. We extend the Gatenby-Gawlinski reaction-diffusion model to derive a system of five coupled reaction-diffusion equations to describe the resulting invasion process. We characterise the spatially homogeneous steady states and carry out a simulation study in one spatial dimension to determine how the tumour develops as we vary the strength of competition between the two phenotypes. We find that overall tumour growth is most extensive when both cell types can stably coexist, since this allows the cells to locally mix and benefit most from the combination of traits. In contrast, when inter-species competition exceeds intra-species competition the populations spatially separate and invasion arrests either: (i) rapidly (matrix-degraders dominate) or (ii) slowly (acid-producers dominate). Overall, our work demonstrates that the spatial and ecological relationship between a heterogeneous population of tumour cells is a key factor in determining their ability to cooperate. Specifically, we predict that tumours in which different phenotypes coexist stably are more invasive than tumours in which phenotypes are spatially separated.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Wu2020-nn" class="col-sm-10"> <div class="title">T-cells produce acidic niches in lymph nodes to suppress their own effector functions</div> <div class="author"> Hao Wu, Veronica Estrella, Matthew Beatty, Dominique Abrahams, Asmaa El-Kenawi, Shonagh Russell, Arig Ibrahim-Hashim, Dario Livio Longo, Yana K Reshetnyak, Anna Moshnikova, Oleg A Andreev, Kimberly Luddy, <em>Mehdi Damaghi</em>, Krithika Kodumudi, Smitha R Pillai, Pedro Enriquez-Navas, Shari Pilon-Thomas, Pawel Swietach, and Robert J Gillies </div> <div class="periodical"> <em>Nat. Commun.</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The acidic pH of tumors profoundly inhibits effector functions of activated CD8 + T-cells. We hypothesize that this is a physiological process in immune regulation, and that it occurs within lymph nodes (LNs), which are likely acidic because of low convective flow and high glucose metabolism. Here we show by in vivo fluorescence and MR imaging, that LN paracortical zones are profoundly acidic. These acidic niches are absent in athymic Nu/Nu and lymphodepleted mice, implicating T-cells in the acidifying process. T-cell glycolysis is inhibited at the low pH observed in LNs. We show that this is due to acid inhibition of monocarboxylate transporters (MCTs), resulting in a negative feedback on glycolytic rate. Importantly, we demonstrate that this acid pH does not hinder initial activation of naïve T-cells by dendritic cells. Thus, we describe an acidic niche within the immune system, and demonstrate its physiological role in regulating T-cell activation.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Sarhadi2020-yn" class="col-sm-10"> <div class="title">Omics integration analyses reveal the early evolution of malignancy in breast cancer</div> <div class="author"> Shamim Sarhadi, Ali Salehzadeh-Yazdi, <em>Mehdi Damaghi</em>, N Zarghami, O Wolkenhauer, and Hedayatollah Hosseini </div> <div class="periodical"> <em>Cancers</em>, Apr 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Background The majority of cancer evolution studies are done on individual-based approaches that neglect population dynamics necessity for the global picture of cancer evolution in each cancer type. Here, we conducted a population-based study in breast cancer to understand the timing of malignancy evolution and its correlation to the genetic evolution of pathological stages. Results In an omics integrative approach, we integrated gene expression and genomic aberration data for pre-invasive (DCIS, early-stage) and post-invasive (IDC, late-stage) samples and investigated the evolutionary role of further genetic changes in late stages compared to the early ones. We found that single genetic alterations (SGAs) and copy number alterations (CNAs) conspire together for the fine-tuning of the operating signaling pathways of tumors in forward and backward evolution manners. The forward evolution applies to new genetic changes that boost the efficiency of selected signaling pathways. The backward evolution, which we detected for CNAs, is a mean to reverse unwanted SGAs of earlier stages. Analyses of the integrated point mutation and gene expression data show that (i) our proposed fine-tuning concept is also applicable in metastasis, and (ii) metastasis diverges from primary tumor sometimes at the DCIS stage. Conclusions Our results indicate that malignant potency of breast tumors is constant over pre and post invasive pathological stages. Indeed, further genetic alterations in later stages do not establish de novo malignancy routes; however, they serve to fine-tune antecedent signaling pathways.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Wu2019-ab" class="col-sm-10"> <div class="title">Lymph nodes inhibit T-cell effector functions locally by establishing acidic niches</div> <div class="author"> Hao Wu, Veronica Estrella, Pedro Enriquez-Navas, Asmaa El-Kenawi, Shonagh Russell, Dominique Abrahams, Arig Ibrahim-Hashim, Dario Longo, Yana Reshetnyak, Kimberly Luddy, <em>Mehdi Damaghi</em>, Smitha Ravindranadhan Pillai, Matthew Beatty, Shari Pilon-Thomas, Pawel Swietach, and Robert J Gillies </div> <div class="periodical"> <em>bioRxiv</em>, Jul 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>AbstractLymph nodes are an essential component of the adaptive immune response where antigen-presenting cells are closely housed with their cognate effector cells. Protection of lymph node resident cells from activated immune cells in such close quarters would need to be robust and reversible. Effector functions of T-cells are profoundly and reversibly inhibited by an acidic microenvironment. The underlying mechanisms of this inhibition are unknown, but may relate to glycolysis, which is obligatory for expression of effector functions. Here, we demonstrate that acidification rapidly and potently inhibits monocarboxylate transporter-dependent lactic acid efflux, which dually inhibits glycolysis by end-product accumulation and by reducing cytoplasmic pH. Based on the robustness of these responses, we propose that acid-evoked T-cell inhibition is physiologically important, and that lymph nodes are a natural site for such modulation. Using multiple imaging techniques, we show that paracortical T-zones of lymph nodes are highly acidic. We further show that T-cells can be activated by dendritic cells at low pH, and their effector functions are restored rapidly upon increasing pH. Thus, we describe a novel physiological mechanism whereby activated T-cells are kept in stasis by acidosis whilst resident in lymph nodes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Strobl2019-rb" class="col-sm-10"> <div class="title">Mix &amp; Match: Phenotypic coexistence as a key facilitator of solid tumour invasion</div> <div class="author"> Maximilian A R Strobl, Andrew L Krause, <em>Mehdi Damaghi</em>, Robert Gillies, Alexander R A Anderson, and Philip K Maini </div> <div class="periodical"> <em>bioRxiv</em>, Aug 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>AbstractInvasion of healthy tissue is a defining feature of malignant tumours. Traditionally, invasion is thought to be driven by cells that have acquired all the necessary traits to overcome the range of biological and physical defences employed by the body. However, in light of the ever-increasing evidence for geno- and phenotypic intra-tumour heterogeneity an alternative hypothesis presents itself: Could invasion be driven by a collection of cells with distinct traits that together facilitate the invasion process? In this paper, we use a mathematical model to assess the feasibility of this hypothesis in the context of acid-mediated invasion. We assume tumour expansion is obstructed by stroma which inhibits growth, and extra-cellular matrix (ECM) which blocks cancer cell movement. Further, we assume that there are two types of cancer cells: i) a glycolytic phenotype which produces acid that kills stromal cells, and ii) a matrix-degrading phenotype that locally remodels the ECM. We extend the Gatenby-Gawlinski reaction-diffusion model to derive a system of five coupled reaction-diffusion equations to describe the resulting invasion process. We characterise the spatially homogeneous steady states and carry out a simulation study in one spatial dimension to determine how the tumour develops as we vary the strength of competition between the two phenotypes. We find that overall tumour growth is most extensive when both cell types can stably coexist, since this allows the cells to locally mix and benefit most from the combination of traits. In contrast, when inter-species competition exceeds intra-species competition the populations spatially separate and invasion arrests either: i) rapidly (matrix-degraders dominate), or ii) slowly (acid-producers dominate). Overall, our work demonstrates that the spatial and ecological relationship between a heterogeneous population of tumour cells is a key factor in determining their ability to cooperate. Specifically, we predict that tumours in which different phenotypes coexist stably are more invasive than tumours in which phenotypes are spatially separated.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Pillai2019-mf" class="col-sm-10"> <div class="title">Causes, consequences, and therapy of tumors acidosis</div> <div class="author"> Smitha R Pillai, <em>Mehdi Damaghi</em>, Yoshinori Marunaka, Enrico Pierluigi Spugnini, Stefano Fais, and Robert J Gillies </div> <div class="periodical"> <em>Cancer Metastasis Rev.</em>, Jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>While cancer is commonly described as “a disease of the genes,” it is also associated with massive metabolic reprogramming that is now accepted as a disease “Hallmark.” This programming is complex and often involves metabolic cooperativity between cancer cells and their surrounding stroma. Indeed, there is emerging clinical evidence that interrupting a cancer’s metabolic program can improve patients’ outcomes. The most commonly observed and well-studied metabolic adaptation in cancers is the fermentation of glucose to lactic acid, even in the presence of oxygen, also known as “aerobic glycolysis” or the “Warburg Effect.” Much has been written about the mechanisms of the Warburg effect, and this remains a topic of great debate. However, herein, we will focus on an important sequela of this metabolic program: the acidification of the tumor microenvironment. Rather than being an epiphenomenon, it is now appreciated that this acidosis is a key player in cancer somatic evolution and progression to malignancy. Adaptation to acidosis induces and selects for malignant behaviors, such as increased invasion and metastasis, chemoresistance, and inhibition of immune surveillance. However, the metabolic reprogramming that occurs during adaptation to acidosis also introduces therapeutic vulnerabilities. Thus, tumor acidosis is a relevant therapeutic target, and we describe herein four approaches to accomplish this: (1) neutralizing acid directly with buffers, (2) targeting metabolic vulnerabilities revealed by acidosis, (3) developing acid-activatable drugs and nanomedicines, and (4) inhibiting metabolic processes responsible for generating acids in the first place.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Pillai2018-ni" class="col-sm-10"> <div class="title">Acid-induced autophagic protein products are stored as adiposomes in breast cancer cells</div> <div class="author"> Smitha R Pillai, Jonathan W Wojtkowiak, Jonathan Nguyen, <em>Mehdi Damaghi</em>, Marilyn M Bui, Timothy Garrett, and Robert J Gillies </div> <div class="periodical"> <em>Cancer Research</em>, Jul 2018 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="Strobl2018-as" class="col-sm-10"> <div class="title">Connecting the microenvironmental niche to treatment response in ovarian cancer</div> <div class="author"> Maximilian Strobl, Matthew Wicker, Vikram Adhikarla, W Andrew Shockey, Eszter Lakatos, Pantea Pooladvand, Ryan Schenk, Linggih Saputro, Chandler Gatenbee, Martijn Koppens, Salvador Cruz García, Robert Wenham, <em>Mehdi Damaghi</em>, and Jill A Gallaher </div> <div class="periodical"> <em>bioRxiv</em>, Oct 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>AbstractOvarian cancer has the highest mortality rate of all gynecologic cancers, which may be attributed to an often late stage diagnosis, when the cancer is already metastatic, and rapid development of treatment resistance. We propose that the metastatic disease could be better characterized by observing interactions within the microenvironmental niche of the primary site that shapes the tumor’s early phenotypic progression. We present a mechanistic mathematical model of ovarian cancer that considers spatial interactions between tumor cells and several key stromal components. We demonstrate how spatial biomarker imaging data from the primary tumor can be analyzed to define a patient-specific microenvironment in the mathematical model. We then show preliminary results, using this model, that demonstrate how differences in the niche composition of a tumor affects phenotypic evolution and treatment response.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Persi2018-vq" class="col-sm-10"> <div class="title">Systems analysis of intracellular pH vulnerabilities for cancer therapy</div> <div class="author"> Erez Persi, Miquel Duran-Frigola, <em>Mehdi Damaghi</em>, William R Roush, Patrick Aloy, John L Cleveland, Robert J Gillies, and Eytan Ruppin </div> <div class="periodical"> <em>Nat. Commun.</em>, Jul 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>A reverse pH gradient is a hallmark of cancer metabolism, manifested by extracellular acidosis and intracellular alkalization. While consequences of extracellular acidosis are known, the roles of intracellular alkalization are incompletely understood. By reconstructing and integrating enzymatic pH-dependent activity profiles into cell-specific genome-scale metabolic models, we develop a computational methodology that explores how intracellular pH (pHi) can modulate metabolism. We show that in silico, alkaline pHi maximizes cancer cell proliferation coupled to increased glycolysis and adaptation to hypoxia (i.e., the Warburg effect), whereas acidic pHi disables these adaptations and compromises tumor cell growth. We then systematically identify metabolic targets (GAPDH and GPI) with predicted amplified anti-cancer effects at acidic pHi, forming a novel therapeutic strategy. Experimental testing of this strategy in breast cancer cells reveals that it is particularly effective against aggressive phenotypes. Hence, this study suggests essential roles of pHi in cancer metabolism and provides a conceptual and computational framework for exploring pHi roles in other biomedical domains.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Damaghi2017-ye" class="col-sm-10"> <div class="title">Phenotypic changes of acid-adapted cancer cells push them toward aggressiveness in their evolution in the tumor microenvironment</div> <div class="author"> <em>Mehdi Damaghi</em>, and Robert Gillies </div> <div class="periodical"> <em>Cell Cycle</em>, Oct 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The inter- and intra-tumoral metabolic phenotypes of tumors are heterogeneous, and this is related to microenvironments that select for increased glycolysis. Increased glycolysis leads to decreased pH, and these local microenvironment effects lead to further selection. Hence, heterogeneity of phenotypes is an indirect consequence of altering microenvironments during carcinogenesis. In early stages of growth, tumors are stratified, with the most aggressive cells developing within the acidic interior of the tumor. However, these cells eventually find themselves at the tumor edge, where they invade into the normal tissue via acid-mediated invasion. We believe acid adaptation during the evolution of cancer cells in their niche is a Rubicon that, once crossed, allows cells to invade into and outcompete normal stromal tissue. In this study, we illustrate some acid-induced phenotypic changes due to acidosis resulting in more aggressiveness and invasiveness of cancer cells.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Saeed-Vafa2017-ez" class="col-sm-10"> <div class="title">Combining radiomics and mathematical modeling to elucidate mechanisms of resistance to immune checkpoint blockade in non-small cell lung cancer</div> <div class="author"> Daryoush Saeed-Vafa, Rafael Bravo, Jamie A Dean, Asmaa El-Kenawi, Nathaniel Mon Père, Maximilian Strobl, Charlie Daniels, Olya Stringfield, <em>Mehdi Damaghi</em>, Ilke Tunali, Liam V Brown, Lee Curtin, Daniel Nichol, Hailee Peck, Robert J Gillies, and Jill A Gallaher </div> <div class="periodical"> <em>bioRxiv</em>, Sep 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>AbstractImmune therapies have shown promise in a number of cancers, and clinical trials using the anti-PD-L1/PD-1 checkpoint inhibitor in lung cancer have been successful for a number of patients. However, some patients either do not respond to the treatment or have cancer recurrence after an initial response. It is not clear which patients might fall into these categories or what mechanisms are responsible for treatment failure. To explore the different underlying biological mechanisms of resistance, we created a spatially explicit mathematical model with a modular framework. This construction enables different potential mechanisms to be turned on and off in order to adjust specific tumor and tissue interactions to match a specific patient’s disease. In parallel, we developed a software suite to identify significant computed tomography (CT) imaging features correlated with outcome using data from an anti-PDL-1 checkpoint inhibitor clinical trial for lung cancer and a tool that extracts these features from both patient CT images and “virtual CT” images created from the cellular density profile of the model. The combination of our two toolkits provides a framework that feeds patient data through an iterative pipeline to identify predictive imaging features associated with outcome, whilst at the same time proposing hypotheses about the underlying resistance mechanisms.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ibrahim-Hashim2017-hw" class="col-sm-10"> <div class="title">Defining Cancer Subpopulations by Adaptive Strategies Rather Than Molecular Properties Provides Novel Insights into Intratumoral Evolution</div> <div class="author"> Arig Ibrahim-Hashim, Mark Robertson-Tessi, Pedro M Enriquez-Navas, <em>Mehdi Damaghi</em>, Yoganand Balagurunathan, Jonathan W Wojtkowiak, Shonagh Russell, Kam Yoonseok, Mark C Lloyd, Marilyn M Bui, Joel S Brown, Alexander R A Anderson, Robert J Gillies, and Robert A Gatenby </div> <div class="periodical"> <em>Cancer Res.</em>, May 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Ongoing intratumoral evolution is apparent in molecular variations among cancer cells from different regions of the same tumor, but genetic data alone provide little insight into environmental selection forces and cellular phenotypic adaptations that govern the underlying Darwinian dynamics. In three spontaneous murine cancers (prostate cancers in TRAMP and PTEN mice, pancreatic cancer in KPC mice), we identified two subpopulations with distinct niche construction adaptive strategies that remained stable in culture: (i) invasive cells that produce an acidic environment via upregulated aerobic glycolysis; and (ii) noninvasive cells that were angiogenic and metabolically near-normal. Darwinian interactions of these subpopulations were investigated in TRAMP prostate cancers. Computer simulations demonstrated invasive, acid-producing (C2) cells maintain a fitness advantage over noninvasive, angiogenic (C3) cells by promoting invasion and reducing efficacy of immune response. Immunohistochemical analysis of untreated tumors confirmed that C2 cells were invariably more abundant than C3 cells. However, the C2 adaptive strategy phenotype incurred a significant cost due to inefficient energy production (i.e., aerobic glycolysis) and depletion of resources for adaptations to an acidic environment. Mathematical model simulations predicted that small perturbations of the microenvironmental extracellular pH (pHe) could invert the cost/benefit ratio of the C2 strategy and select for C3 cells. In vivo, 200 mmol/L NaHCO3 added to the drinking water of 4-week-old TRAMP mice increased the intraprostatic pHe by 0.2 units and promoted proliferation of noninvasive C3 cells, which remained confined within the ducts so that primary cancer did not develop. A 0.2 pHe increase in established tumors increased the fraction of C3 cells and signficantly diminished growth of primary and metastatic tumors. In an experimental tumor construct, MCF7 and MDA-MB-231 breast cancer cells were coinjected into the mammary fat pad of SCID mice. C2-like MDA-MB-231 cells dominated in untreated animals, but C3-like MCF7 cells were selected and tumor growth slowed when intratumoral pHe was increased. Overall, our data support the use of mathematical modeling of intratumoral Darwinian interactions of environmental selection forces and cancer cell adaptive strategies. These models allow the tumor to be steered into a less invasive pathway through the application of small but selective biological force. Cancer Res; 77(9); 2242-54. ©2017 AACR.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div id="Damaghi2016-zh" class="col-sm-10"> <div class="title">Lysosomal protein relocation as an adaptation mechanism to extracellular acidosis</div> <div class="author"> <em>Mehdi Damaghi</em>, and Robert J Gillies </div> <div class="periodical"> <em>Cell Cycle</em>, Jul 2016 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div id="Damaghi2015-mc" class="col-sm-10"> <div class="title">Chronic acidosis in the tumour microenvironment selects for overexpression of LAMP2 in the plasma membrane</div> <div class="author"> <em>Mehdi Damaghi</em>, Narges K Tafreshi, Mark C Lloyd, Robert Sprung, Veronica Estrella, Jonathan W Wojtkowiak, David L Morse, John M Koomen, Marilyn M Bui, Robert A Gatenby, and Robert J Gillies </div> <div class="periodical"> <em>Nat. Commun.</em>, Dec 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Early cancers are avascular and hence, profoundly acidic. Pre-malignant cells must adapt to acidosis to thrive in this hostile microenvironment. Here, we investigate MCF-7 cells that are adapted to grow in acidic conditions using SILAC proteomics and we reveal a significant upregulation of lysosomal proteins. Prominent among these is LAMP2 that functions to protect lysosomal membranes from acid proteolysis. LAMP2 upregulation by acidosis is confirmed both in vitro and in vivo. Furthermore, we show that the depletion of LAMP2 is sufficient to increase acidosis-mediated toxicity. In breast cancer patient samples, there is a high correlation of LAMP2 mRNA and protein expression with progression. We also observe that LAMP2 is located at the plasma membrane in clinical samples and this redistribution is acid-induced in vitro. Our findings suggest a potential adaptive mechanism, wherein cells chronically exposed to an acidic environment translocate lysosomal proteins to their surface, thus protecting the plasmalemma from acid-induced hydrolysis.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Damaghi2013-mb" class="col-sm-10"> <div class="title">Characterizing the Functional and Folding Mechanism of β-barrel Transmembrane Proteins Using Atomic Force Microscope</div> <div class="author"> <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em></em> Dec 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Abstract (EN) Single-molecule force spectroscopy (SMFS) is a unique approach to study the mechanical unfolding of proteins. SMFS unfolding experiments yield insight into how interactions stabilize a protein and guide its unfolding and refolding pathways. In contrast to various water-soluble proteins whose unfolding and refolding patterns have been characterized, only α-helical membrane proteins have been probed by SMFS. It was shown that α-helical membrane proteins unfold via many intermediates; this differs from the two-state unfolding process usually observed in water-soluble proteins. In membrane proteins, upon mechanically pulling the peptide end of the protein, single and grouped α-helices and polypeptide loops unfold in steps until the entire protein is unfolded. Whether the α-helices and loops unfold individually or cooperatively to form an unfolding intermediate depends on the interactions established within the membrane protein and the membrane. Each unfolding event relates to an unfolding intermediate with the sequence of these intermediates defining the unfolding pathway of the protein. β-barrel-forming membrane proteins are the second major group of membrane proteins and have not yet been studied by SMFS. To fill this void this study was designed to characterize interactions, unfolding, and refolding of the β-barrel forming outermembrane protein G (OmpG). Folding of transmembrane proteins, despite the important part these proteins play in every biological process in a cell, is studied in only a few examples. Of those only a handful were β-stranded membrane proteins (Tamm et al., 2004; Kleinschmidt et al., 2006 …</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Damaghi2013-zr" class="col-sm-10"> <div class="title">pH sensing and regulation in cancer</div> <div class="author"> <em>Mehdi Damaghi</em>, Jonathan W Wojtkowiak, and Robert J Gillies </div> <div class="periodical"> <em>Front. Physiol.</em>, Dec 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cells maintain intracellular pH (pHi) within a narrow range (7.1-7.2) by controlling membrane proton pumps and transporters whose activity is set by intra-cytoplasmic pH sensors. These sensors have the ability to recognize and induce cellular responses to maintain the pHi, often at the expense of acidifying the extracellular pH. In turn, extracellular acidification impacts cells via specific acid-sensing ion channels (ASICs) and proton-sensing G-protein coupled receptors (GPCRs). In this review, we will discuss some of the major players in proton sensing at the plasma membrane and their downstream consequences in cancer cells and how these pH-mediated changes affect processes such as migration and metastasis. The complex mechanisms by which they transduce acid pH signals to the cytoplasm and nucleus are not well understood. However, there is evidence that expression of proton-sensing GPCRs such as GPR4, TDAG8, and OGR1 can regulate aspects of tumorigenesis and invasion, including cofilin and talin regulated actin (de-)polymerization. Major mechanisms for maintenance of pHi homeostasis include monocarboxylate, bicarbonate, and proton transporters. Notably, there is little evidence suggesting a link between their activities and those of the extracellular H(+)-sensors, suggesting a mechanistic disconnect between intra- and extracellular pH. Understanding the mechanisms of pH sensing and regulation may lead to novel and informed therapeutic strategies that can target acidosis, a common physical hallmark of solid tumors.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div id="Damaghi2011-vj" class="col-sm-10"> <div class="title">One β hairpin follows the other: Exploring refolding pathways and kinetics of the transmembrane β-barrel protein OmpG</div> <div class="author"> <em>Mehdi Damaghi</em>, Stefan Köster, Christian A Bippes, Özkan Yildiz, and Daniel J Müller </div> <div class="periodical"> <em>Angew. Chem. Weinheim Bergstr. Ger.</em>, Aug 2011 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Damaghi2010-ia" class="col-sm-10"> <div class="title">pH-dependent interactions guide the folding and gate the transmembrane pore of the beta-barrel membrane protein OmpG</div> <div class="author"> <em>Mehdi Damaghi</em>, Christian Bippes, Stefan Köster, Ozkan Yildiz, Stefania A Mari, Werner Kühlbrandt, and Daniel J Muller </div> <div class="periodical"> <em>J. Mol. Biol.</em>, Apr 2010 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The physical interactions that switch the functional state of membrane proteins are poorly understood. Previously, the pH-gating conformations of the beta-barrel forming outer membrane protein G (OmpG) from Escherichia coli have been solved. When the pH changes from neutral to acidic the flexible extracellular loop L6 folds into and closes the OmpG pore. Here, we used single-molecule force spectroscopy to structurally localize and quantify the interactions that are associated with the pH-dependent closure. At acidic pH, we detected a pH-dependent interaction at loop L6. This interaction changed the (un)folding of loop L6 and of beta-strands 11 and 12, which connect loop L6. All other interactions detected within OmpG were unaffected by changes in pH. These results provide a quantitative and mechanistic explanation of how pH-dependent interactions change the folding of a peptide loop to gate the transmembrane pore. They further demonstrate how the stability of OmpG is optimized so that pH changes modify only those interactions necessary to gate the transmembrane pore.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Damaghi2010-wb" class="col-sm-10"> <div class="title">Dual energy landscape: the functional state of the β-barrel outer membrane protein G molds its unfolding energy landscape</div> <div class="author"> <em>Mehdi Damaghi</em>, K Tanuj Sapra, Stefan Köster, Özkan Yildiz, Werner Kühlbrandt, and Daniel J Muller </div> <div class="periodical"> <em>Proteomics</em>, Dec 2010 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We applied dynamic single-molecule force spectroscopy to quantify the parameters (free energy of activation and distance of the transition state from the folded state) characterizing the energy barriers in the unfolding energy landscape of the outer membrane protein G (OmpG) from Escherichia coli. The pH-dependent functional switching of OmpG directs the protein along different regions on the unfolding energy landscape. The two functional states of OmpG take the same unfolding pathway during the sequential unfolding of β-hairpins I-IV. After the initial unfolding events, the unfolding pathways diverge. In the open state, the unfolding of β-hairpin V in one step precedes the unfolding of β-hairpin VI. In the closed state, β-hairpin V and β-strand S11 with a part of extracellular loop L6 unfold cooperatively, and subsequently β-strand S12 unfolds with the remaining loop L6. These two unfolding pathways in the open and closed states join again in the last unfolding step of β-hairpin VII. Also, the conformational change from the open to the closed state witnesses a rigidified extracellular gating loop L6. Thus, a change in the conformational state of OmpG not only bifurcates its unfolding pathways but also tunes its mechanical properties for optimum function.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"><li> <div class="row"> <div id="Tanuj-Sapra2009-pq" class="col-sm-10"> <div class="title">One b hairpin after the other: exploring mechanical unfolding pathways of the transmembrane b-barrel protein OmpG</div> <div class="author"> K Tanuj Sapra, <em>Mehdi Damaghi</em>, Stefan Köster, O Yildiz, Werner Kühlbrandt, and D J Müller </div> <div class="periodical"> <em>Angewandte Chemie International Edition</em>, Dec 2009 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"><li> <div class="row"> <div id="damaghi2008a" class="col-sm-10"> <div class="title">Dendrosomes as novel gene porters-III</div> <div class="author"> Majid Sadeghizadeh, Bijan Ranjbar, <em>Mehdi Damaghi</em>, Leila Khaki, Mohammad N Sarbolouki, Farhood Najafi, Simak Parsaee, Abed-Ali Ziaee, Mohammad Massumi, Werner Lubitz, Paul Kudela, Susan Paukner, and Ali Karami </div> <div class="periodical"> <em>Journal of Chemical Technology &amp; Biotechnology</em>, Dec 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/https://doi.org/10.1002/jctb.1891" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>It was previously reported that dendrosomes, i.e. neutral, biodegradable, covalent or self-assembled, hyperbranched, spheroidal nano-particles with a size ranging from 15 to 100 nm, provide a convenient and efficient means of gene delivery into various kinds of cells such as human hepatoma and kidney cells as well as animal models. RESULTS: New studies via circular dichroism show that hydrophilic and amphipathic dendrosomes either do not affect the DNA structure or moderately transform it from B- to A-conformation. Gene delivery into human liver, kidney, and endothelial cells as well as other animal cells like Bowes, U-937, Raw, CCRF-CEM, MOLT-4, K562, Huh-7 and VERO reveal that the genes are efficiently expressed and in comparison with other gene porters like Lipofectin or bacterial ghosts, do quite well. It is also shown that dendrosomes are able to deliver genes into cells like endothelials that are usually hard to transfect. Cell culture experiments as well as intraperitoneal/intradermal injections of dendrosomes into mice establish their nontoxicity (up to 2.5 mg kg−1 of animal weight in the latter case). Studies on immunization of BALB/c mice using conventional adjuvants such as aluminium phosphate, CpG motif and one of the dendrosomes, indicate that the latter leads to the mildest initial response development while exceeding them afterwards. CONCLUSION: CD studies reveal that, owing to the neutrality of dendrosomes, formation of Den/DNA complexes is accompanied by slight structural modifications of DNA cell culture, and animal studies reveal that dendrosomes are inert, non-toxic and highly efficient gene porters that perform at extremely low doses. In comparison with bacterial ghosts and some common porters, they are efficient in delivery of genes into animals and a variety of cells including those that are usually hard to transfect.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 CEEL : Cancer Ecology &amp; Evolution Lab. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>