<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | CEEL: Cancer Ecology &amp; Evolution Lab </title> <meta name="author" content="CEEL : Cancer Ecology &amp; Evolution Lab"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://sbu-damaghi-ceel.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> CEEL: Cancer Ecology &amp; Evolution Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">search <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Xiao2024-ty" class="col-sm-8"> <div class="title">Eco-evolutionary guided pathomic analysis to predict DCIS upstaging</div> <div class="author"> Yujie Xiao, Manal Elmasry, Ji Dong K Bai, Andrew Chen, Yuzhu Chen, Brooke Jackson, Joseph O Johnson, Robert J Gillies, Prateek Prasanna, Chao Chen, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>bioRxivorg</em>, Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cancers evolve in a dynamic ecosystem. Thus, characterizing cancer’s ecological dynamics is crucial to understanding cancer evolution and can lead to discovering novel biomarkers to predict disease progression. Ductal carcinoma in situ (DCIS) is an early-stage breast cancer characterized by abnormal epithelial cell growth confined within the milk ducts. In this study, we show that ecological habitat analysis of hypoxia and acidosis biomarkers can significantly improve prediction of DCIS upstaging. First, we developed a novel eco-evolutionary designed approach to define habitats in the tumor intraductal microenvironment based on oxygen diffusion distance. Then, we identify cancer cells with metabolic phenotypes attributed to their habitat conditions, such as the expression of CA9 indicating hypoxia responding phenotype, and LAMP2b indicating the acid adaptation. Traditionally these markers have shown limited predictive capabilities for DCIS upstaging, if any. However, when analyzed from an ecological perspective, their power to differentiate between pure DCIS and upstaged DCIS increased significantly. Second, using eco-evolutionary guided computational and digital pathology techniques, we discovered distinct niches with spatial patterns of these biomarkers and used the distribution of such niches to predict patient upstaging. The niches patterns were characterized by pattern analysis of both cellular and spatial features. With a 5-fold validation on the biopsy cohort, we trained a random forest classifier to achieve the area under curve (AUC) of 0.74. Our results affirm the importance of using eco-evolutionary-designed approaches in biomarkers discovery studies in the era of digital pathology by demonstrating the role of tumor ecological habitats and niches.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Song2024-zc" class="col-sm-8"> <div class="title">Human and bats genome robustness under COSMIC mutational signatures</div> <div class="author"> Joon-Hyun Song, Ying Zeng, Liliana M Dávalos, Thomas MacCarthy, Mani Larijani, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>bioRxivorg</em>, Sep 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Han2024-bp" class="col-sm-8"> <div class="title">Ecological and evolutionary dynamics to design and improve ovarian cancer treatment</div> <div class="author"> Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>Clin. Transl. Med.</em>, Sep 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP-ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell’s plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. KEY POINTS/HIGHLIGHTS: Tumours are ecosystems in which cancer and non-cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco-evolutionarily designed therapies should consider cancer cell plasticity and patient-specific characteristics to improve clinical outcome and prevent relapse.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Strobl2024-zb" class="col-sm-8"> <div class="title">To modulate or to skip: De-escalating PARP inhibitor maintenance therapy in ovarian cancer using adaptive therapy</div> <div class="author"> Maximilian A R Strobl, Alexandra L Martin, Jeffrey West, Jill Gallaher, Mark Robertson-Tessi, Robert Gatenby, Robert Wenham, Philip K Maini, <em>Mehdi Damaghi</em>, and Alexander R A Anderson </div> <div class="periodical"> <em>Cell Syst.</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Toxicity and emerging drug resistance pose important challenges in poly-adenosine ribose polymerase inhibitor (PARPi) maintenance therapy of ovarian cancer. We propose that adaptive therapy, which dynamically reduces treatment based on the tumor dynamics, might alleviate both issues. Utilizing in vitro time-lapse microscopy and stepwise model selection, we calibrate and validate a differential equation mathematical model, which we leverage to test different plausible adaptive treatment schedules. Our model indicates that adjusting the dosage, rather than skipping treatments, is more effective at reducing drug use while maintaining efficacy due to a delay in cell kill and a diminishing dose-response relationship. In vivo pilot experiments confirm this conclusion. Although our focus is toxicity mitigation, reducing drug use may also delay resistance. This study enhances our understanding of PARPi treatment scheduling and illustrates the first steps in developing adaptive therapies for new treatment settings. A record of this paper’s transparent peer review process is included in the supplemental information.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Song2024-dy" class="col-sm-8"> <div class="title">Evolvability of cancer-associated genes under APOBEC3A/B selection</div> <div class="author"> Joon-Hyun Song, Liliana M Dávalos, Thomas MacCarthy, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>iScience</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Evolvability is an emergent hallmark of cancer that depends on intra-tumor heterogeneity and genetic variation. Mutations generated by APOBEC3 contribute to genetic variation and tumor evolvability. However, the influence of APOBEC3 on the evolvability of the genome and its differential impact on cancer genes versus non-cancer genes remains unclear. Analyzing over 40,000 human protein-coding transcripts, we identified distinct distribution patterns of APOBEC3A/B TC motifs between cancer and non-cancer genes, suggesting unique associations with cancer. Studying a bat species with numerous APOBEC3 genes, we found distinct motif patterns in orthologs of cancer genes compared to non-cancer genes, as in humans, suggesting APOBEC3 evolution to reduce impacts on the genome rather than the converse. Simulations confirmed that APOBEC3-induced heterogeneity enhances cancer evolution through bimodal patterns of mutations in certain classes of genes. Our results suggest the bimodal distribution of APOBEC-induced mutations can significantly increase cancer heterogeneity.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Obeid2024-ju" class="col-sm-8"> <div class="title">Metabolic adaptation and epigenetic modulations: Unraveling tumor plasticity under variable tumor microenvironment</div> <div class="author"> Jowana Obeid, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em></em> Jan 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The tumor microenvironment (TME) variation plays a crucial role in cancer initiation, progression, and evolution. Tumor plasticity, defined as the ability of cancer cells to adapt their variable TME with no need for mutations or genetic changes, has emerged as a hallmark of cancer driving tumor evolution. This chapter discusses the metabolic adaptations as part of tumor plasticity to variable TME that occur in solid tumors, affecting the tumor evolutionary trajectories. It highlights the role of epigenetic modifications in adaptation to variable TME, including DNA methylation, chromatin modifications, and noncoding RNA regulation. Additionally, it explores how the TME conditions cause reprograming cellular metabolism, including glucose, amino acid, and lipid metabolism, leading to major rewiring of transcriptional networks and accumulation of reactive oxygen species (ROS) that promote tumor progression. The interplay …</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Poonja2023-do" class="col-sm-8"> <div class="title">Dynamics of fibril collagen remodeling by tumor cells: A model of tumor-associated collagen signatures</div> <div class="author"> Sharan Poonja, Ana Forero Pinto, Mark C Lloyd, <em>Mehdi Damaghi</em>, and K Rejniak </div> <div class="periodical"> <em>Cells</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Many solid tumors are characterized by a dense extracellular matrix (ECM) composed of various ECM fibril proteins. These proteins provide structural support and a biological context for the residing cells. The reciprocal interactions between growing and migrating tumor cells and the surrounding stroma result in dynamic changes in the ECM architecture and its properties. With the use of advanced imaging techniques, several specific patterns in the collagen surrounding the breast tumor have been identified in both tumor murine models and clinical histology images. These tumor-associated collagen signatures (TACS) include loosely organized fibrils far from the tumor and fibrils aligned either parallel or perpendicular to tumor colonies. They are correlated with tumor behavior, such as benign growth or invasive migration. However, it is not fully understood how one specific fibril pattern can be dynamically remodeled to form another alignment. Here, we present a novel multi-cellular lattice-free (MultiCell-LF) agent-based model of ECM that, in contrast to static histology images, can simulate dynamic changes between TACSs. This model allowed us to identify the rules of cell–ECM physical interplay and feedback that guided the emergence and transition among various TACSs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Song2023-ot" class="col-sm-8"> <div class="title">Evolvability of cancer-associated genes under APOBEC3A/B selection</div> <div class="author"> Joon-Hyun Song, Liliana M Dávalos, Thomas MacCarthy, and <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em>bioRxiv</em>, Aug 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Evolvability is an emergent hallmark of cancer that depends on intra-tumor heterogeneity and, ultimately, genetic variation. Mutations generated by APOBEC3 cytidine deaminases can contribute to genetic variation and the consequences of APOBEC activation differ depending on the stage of cancer, with the most significant impact observed during the early stages. However, how APOBEC activity shapes evolutionary patterns of genes in the host genome and differential impacts on cancer-associated and non-cancer genes remain unclear. Analyzing over 40,000 human protein-coding transcripts, we identified distinct distribution patterns of APOBEC3A/B TC motifs between cancer-related genes and controls, suggesting unique associations with cancer. Studying a bat species with many more APOBEC3 genes, we found diverse motif patterns in orthologs of cancer genes compared to controls, similar to humans and suggesting APOBEC evolution to reduce impacts on the genome rather than the converse. Simulations confirmed that APOBEC-induced heterogeneity enhances cancer evolution, shaping clonal dynamics through bimodal introduction of mutations in certain classes of genes. Our results suggest that a major consequence of the bimodal distribution of APOBEC affects greater cancer heterogeneity. Highlights ### Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Strobl2023-rp" class="col-sm-8"> <div class="title">Adaptive therapy for ovarian cancer: An integrated approach to PARP inhibitor scheduling</div> <div class="author"> Maximilian Strobl, Alexandra L Martin, Jeffrey West, Jill Gallaher, Mark Robertson-Tessi, Robert Gatenby, Robert Wenham, Philip Maini, <em>Mehdi Damaghi</em>, and Alexander Anderson </div> <div class="periodical"> <em>bioRxivorg</em>, Mar 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Toxicity and emerging drug resistance are important challenges in PARP inhibitor (PARPi) treatment of ovarian cancer. Recent research has shown that evolutionary-inspired treatment algorithms which adapt treatment to the tumorâ€™s treatment response (adaptive therapy) can help to mitigate both. Here, we present a first step in developing an adaptive therapy protocol for PARPi treatment by combining mathematical modelling and wet-lab experiments to characterize the cell population dynamics under different PARPi schedules. Using data from in vitro Incucyte Zoom time-lapse microscopy experiments and a step-wise model selection process we derive a calibrated and validated ordinary differential equation model, which we then use to test different plausible adaptive treatment schedules. Our model can accurately predict the in vitro treatment dynamics, even to new schedules, and suggests that treatment modifications need to be carefully timed, or one risks losing control over tumour growth, even in the absence of any resistance. This is because our model predicts that multiple rounds of cell division are required for cells to acquire sufficient DNA damage to induce apoptosis. As a result, adaptive therapy algorithms that modulate treatment but never completely withdraw it are predicted to perform better in this setting than strategies based on treatment interruptions. Pilot experiments in vivo confirm this conclusion. Overall, this study contributes to a better understanding of the impact of scheduling on treatment outcome for PARPis and showcases some of the challenges involved in developing adaptive therapies for new treatment settings.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Damaghi2022-tu" class="col-sm-8"> <div class="title">Selection for a glycolytic phenotype</div> <div class="author"> <em>Mehdi Damaghi</em> </div> <div class="periodical"> <em></em> May 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="damaghi2008a" class="col-sm-8"> <div class="title">Dendrosomes as novel gene porters-III</div> <div class="author"> Majid Sadeghizadeh, Bijan Ranjbar, <em>Mehdi Damaghi</em>, Leila Khaki, Mohammad N Sarbolouki, Farhood Najafi, Simak Parsaee, Abed-Ali Ziaee, Mohammad Massumi, Werner Lubitz, Paul Kudela, Susan Paukner, and Ali Karami </div> <div class="periodical"> <em>Journal of Chemical Technology &amp; Biotechnology</em>, May 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/https://doi.org/10.1002/jctb.1891" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>It was previously reported that dendrosomes, i.e. neutral, biodegradable, covalent or self-assembled, hyperbranched, spheroidal nano-particles with a size ranging from 15 to 100 nm, provide a convenient and efficient means of gene delivery into various kinds of cells such as human hepatoma and kidney cells as well as animal models. RESULTS: New studies via circular dichroism show that hydrophilic and amphipathic dendrosomes either do not affect the DNA structure or moderately transform it from B- to A-conformation. Gene delivery into human liver, kidney, and endothelial cells as well as other animal cells like Bowes, U-937, Raw, CCRF-CEM, MOLT-4, K562, Huh-7 and VERO reveal that the genes are efficiently expressed and in comparison with other gene porters like Lipofectin or bacterial ghosts, do quite well. It is also shown that dendrosomes are able to deliver genes into cells like endothelials that are usually hard to transfect. Cell culture experiments as well as intraperitoneal/intradermal injections of dendrosomes into mice establish their nontoxicity (up to 2.5 mg kg−1 of animal weight in the latter case). Studies on immunization of BALB/c mice using conventional adjuvants such as aluminium phosphate, CpG motif and one of the dendrosomes, indicate that the latter leads to the mildest initial response development while exceeding them afterwards. CONCLUSION: CD studies reveal that, owing to the neutrality of dendrosomes, formation of Den/DNA complexes is accompanied by slight structural modifications of DNA cell culture, and animal studies reveal that dendrosomes are inert, non-toxic and highly efficient gene porters that perform at extremely low doses. In comparison with bacterial ghosts and some common porters, they are efficient in delivery of genes into animals and a variety of cells including those that are usually hard to transfect.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 CEEL : Cancer Ecology &amp; Evolution Lab. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>